keras_nlp/models/bert/bert_presets.py

```diff
@@ -103,6 +127,14 @@
             "Base size of BERT where all input is lowercased. "
             "Trained on English Wikipedia + BooksCorpus."
         ),
+        "metadata":{
+            "description": (
+                "Base size of BERT where all input is lowercased. "
```

Comment:
We have some slightly more informative descriptions on [keras.io ](https://keras.io/api/keras_nlp/models/) currently.

E.g. this one is

```
12-layer BERT model where all input is lowercased.
Trained on English Wikipedia + BooksCorpus.
```

Could we update to the text that is on keras.io and use that as a consistent style?